src/renderer/widgets/SendAssetModal/model/send-asset.ts

```diff
@@ -15,17 +52,175 @@ const getConfigFx = attach({ effect: xcmModel.effects.getConfigFx });
 const saveConfigFx = attach({ effect: xcmModel.effects.saveConfigFx });
 const fetchConfigFx = attach({ effect: xcmModel.effects.fetchConfigFx });
 
+const getParaIdFx = createEffect(async (api: ApiPromise): Promise<number | null> => {
+  try {
+    return await getParachainId(api);
```

Comment:
When returning Promise `await` is not necessary
```suggestion
    return getParachainId(api);
```